README.md

```diff
@@ -0,0 +1,343 @@
+    </a>
+    <a href="http://mypy-lang.org/">
+        <img alt="Checked with MyPy" src="https://camo.githubusercontent.com/34b3a249cd6502d0a521ab2f42c8830b7cfd03fa/687474703a2f2f7777772e6d7970792d6c616e672e6f72672f7374617469632f6d7970795f62616467652e737667">
+    </a>
+    <a href="https://haystack.deepset.ai/docs/intromd">
+        <img alt="Documentation" src="https://img.shields.io/website/http/haystack.deepset.ai/docs/intromd.svg?down_color=red&down_message=offline&up_message=online">
+    </a>
+    <a href="https://github.com/deepset-ai/haystack/releases">
+        <img alt="Release" src="https://img.shields.io/github/release/deepset-ai/haystack">
+    </a>
+    <a href="https://github.com/deepset-ai/haystack/blob/master/LICENSE">
+        <img alt="License" src="https://img.shields.io/github/license/deepset-ai/haystack.svg?color=blue">
+    </a>
+        </a>
+    <a href="https://github.com/deepset-ai/haystack/commits/master">
+        <img alt="Last commit" src="https://img.shields.io/github/last-commit/deepset-ai/haystack">
+    </a>
+</p>
+
+Haystack is an end-to-end framework for Question Answering & Neural search enabling you to ...
+
+... **ask questions** in natural language and find granular answers in your own documents.  
+... **do semantic document search** and retrieve more relevant documents for your search queries.  
+... **search at scale** through millions of documents.  
+... **use off-the-shelf models or fine-tune** them to your own domain.  
+... **evaluate, benchmark and continuously improve** your models via user feedback.  
+... **improve chat bots** by leveraging existing knowledge bases for the long tail of queries.  
+... **automate processes** by automatically applying a list of questions to new documents and using the extracted answers.  
+
+|  |  |
+|-|-|
+| :ledger: [Docs](https://haystack.deepset.ai/docs/intromd) | Usage, Guides, API documentation ...|
+| :computer: [Installation](https://github.com/deepset-ai/haystack/#installation) | How to install |
+| :art: [Key components](https://github.com/deepset-ai/haystack/#key-components) | Overview of core concepts |
+| :eyes: [Quick Tour](https://github.com/deepset-ai/haystack/#quick-tour) | Basic explanation of concepts, options and usage |
+| :mortar_board: [Tutorials](https://github.com/deepset-ai/haystack/#tutorials) | Jupyter/Colab Notebooks & Scripts |
+| :bar_chart: [Benchmarks](https://haystack.deepset.ai/bm/benchmarks) | Speed & Accuracy of Retriever, Readers ... |
+| :telescope: [Roadmap](https://haystack.deepset.ai/en/docs/roadmapmd) | Public roadmap of Haystack |
+| :heart: [Contributing](https://github.com/deepset-ai/haystack/#heart-contributing) | We welcome all contributions! |
+
+## Core Features
+
+-   **Latest models**: Utilize all latest transformer based models (BERT, RoBERTa, MiniLM ...) for extractive QA, generative QA and document retrieval.
```

Comment:
I personally prefer the format (e.g. BERT, RoBERTa, MiniLM). The ... makes the section look to me like it is still a draft. Same goes for the other ... below